benchmarks/test_sharded_parity.py

```diff
@@ -115,6 +115,28 @@ def test_ddp_sharded_plugin_correctness_amp_multi_gpu_ddp(tmpdir, args=None):
     )
 
 
+@pytest.mark.skipif(not _FAIRSCALE_AVAILABLE, reason="Fairscale is not available")
+@pytest.mark.skipif(torch.cuda.device_count() < 2, reason="test requires multi-GPU machine")
+@pytest.mark.skipif(
+    not os.getenv("PL_RUNNING_SPECIAL_TESTS", '0') == '1', reason="test should be run outside of pytest"
+)
+@DDPLauncher.run("--accelerator ddp --gpus 2  --precision 16")
```

Comment:
Wow ! You might be the first one to add a `DDPLauncher` for a test :)

Also, update the skipif with our RunIf

@RunIf(special=True, fairscale=True, min_gpus=2)

Reply:
Adapted